### DISCUSSION TITLE
faulty version of the OpenSSL library

### DISCUSSION THREAD

**Discussion by josquinet** [airsdk/Adobe-Runtime-Support#2247] [General]
Good morning. I am trying to upload an aab file to google play console. (App made with animate). It tells me this error:

"Your application uses a faulty version of the OpenSSL library, which may cause the application to crash. Please update to a fixed version of OpenSSL.
Defective versions:
OpenSSL 1.1.1d"

Later he tells me this:
"contact SDK vendors to download the patch and apply it to your project"

Do I have to ask here? Many thanks

**Comment by shaymolho** [comment]
Please use latest air sdk
https://airsdk.harman.com/download/33.1.1.935
